x11-misc/gmrun/gmrun-0.9.2-r2.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit autotools eutils
+
+DESCRIPTION="A GTK-2 based launcher box with bash style auto completion!"
+HOMEPAGE="https://sourceforge.net/projects/gmrun/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-1"
+SLOT="0"
+KEYWORDS="~amd64 ~mips ~ppc ~x86"
+
+RDEPEND="
+	dev-libs/glib:2
+	dev-libs/popt
+	x11-libs/gtk+:2
+"
+DEPEND="
+	${RDEPEND}
+	elibc_glibc? ( >=sys-libs/glibc-2.10 )
+	sys-apps/sed
+	virtual/pkgconfig
+"
+
+src_prepare() {
+	epatch \
+		"${FILESDIR}"/${P}-gcc43.patch \
+		"${FILESDIR}"/${P}-gcc6-pie.patch \
+		"${FILESDIR}"/${P}-sysconfdir.patch \
+		"${FILESDIR}"/${P}-glibc210.patch \
+		"${FILESDIR}"/${P}-stlport.patch
+
+	eapply_user
+	eautoreconf
+}
+
+src_install() {
```

Comment:
it is default behaviour, you do not need to declare `src_install()` at all.

Reply:
removed src_install